.vscode/cspell.json

```diff
@@ -11,6 +11,8 @@
     "*.exe",
     "*.a",
     "*.lib",
+    "*.yaml",  
+    "**/libcurl-stress-test/README.md", 
```

Comment:
We should try not be disable cspell on markdown text files, like this. Either we fix the spelling errors, or if there are false positives, please add them to the dictionary.

Reply:
IIRC there may be a way to add cspell words to markdown files, which allows you to add them to a local dictionary instead of modifying the master dictionary.

Reply:
Yep, there is! :)
You can add them as a comment on the top, as metadata (which isn't visible), like so:
https://raw.githubusercontent.com/Azure/azure-sdk-for-cpp/86aa9b4b0cfdef2af103ed2330064079ada68773/doc/DistributedTracing.md